rqpy/plotting/_iv_didv_tools_plotting.py

```diff
@@ -149,8 +149,8 @@ def _plot_rload_rn_qetbias(IVanalysisOBJ, lgcsave, xlims_rl, ylims_rl,
     )
     axes[0].grid(True, linestyle = 'dashed')
     axes[0].set_title('Rload vs Vbias', fontsize = 14)
-    axes[0].set_ylabel(r'$R_ℓ$ [mΩ]', fontsize = 14)
-    axes[0].set_xlabel(r'$V_{bias}$ [μV]', fontsize = 14)
+    axes[0].set_ylabel(r'$R_\ell \, [\mathrm{m}\Omega]$', fontsize = 14)
+    axes[0].set_xlabel(r'$V_\text{bias} \, [\mu\mathrm{V}]$', fontsize = 14)
```

Comment:
The `\text` function I don't think is supported in matplotlib LaTeX
```suggestion
    axes[0].set_xlabel(r'$V_\mathrm{bias} \, [\mu\mathrm{V}]$', fontsize = 14)
```